### ISSUE TITLE
Cannot use shovel with flashlight on in v1.1.1

### ISSUE THREAD

**Issue by Hu-Myo** [Shaosil/LethalCompanyMods-FlashlightFix#5]
I don't know why, but this happened after updating to that version and every time I swung the shovel with the flashlight on F, something was written to the console and I couldn't use the shovel, interact with it, or flip item slots.
I'm not sure if it conflicts with any other mods, but this is the only one I've found that involves the flashlight, so I'm reporting it just in case.

Issue Version: v1.1.1
List of mods used
```
BepInEx - BepInExPack-5.4.2100
Rune580-LethalCompany_InputUtils-0.6.3
Sligili - HDLethalCompany-1.5.6
Nyxchrono-DoorBreach-1.1.0
AinaVT-LethalConfig-1.3.4
Ozone-Runtime_Netcode_Patcher-0.2.5
ShaosilGaming-FlashlightFix-1.1.1
Evaisa-HookGenPatcher-0.0.5
Hamunii-JetpackFallFix-2.0.1
Zaggy1024-PathfindingLagFix-1.2.1
CTMods-ChatCommands-1.1.9
Kaguya-SpectatorChat-1.1.1
LeKAKiD-FontPatcher-1.2.3
alexanderjoe-LC_Symphony-1.3.0
fumiko-CullFactory-0.9.0
Um_no-QualityCompany-1.3.6
alexanderjoe-BiggeryBattery-1.0.2
mrov-OrbitRecharge-0.0.5
Evaisa-LethalLib-0.14.2
alexandre_v1-LandMinePlacer-1.0.1
AMRV-ConfigurableCompany-2.6.0
AMRV-LethalCompanyVariables-1.9.0
LethalAPI - LethalAPI_Terminal-1.0.1
mattymatty-LobbyControl-2.2.3
readthisifbad-SnatchinBracken-1.4.0
Hardy-LCMaxSoundsFix-1.1.0
```
This is the log of the problem I was having
[LogOutput.log](https://github.com/Shaosil/LethalCompanyMods-FlashlightFix/files/14443123/LogOutput.log)

How to implement this issue

Enter a facility with a pro flashlight and shovel and turn on the light by pressing the <key that keys the assigned light> key
You can't interact with the shovel or objects in this state

**Comment by Hu-Myo** [comment]
Video of this issue implemented
https://youtu.be/-gT6HqR3suE

I will also post the logs generated from this video
[LogOutput.log](https://github.com/Shaosil/LethalCompanyMods-FlashlightFix/files/14443225/LogOutput.log)

**Comment by Hu-Myo** [comment]
A little more research
It seems to only happen when playing after installing only `FlashlightFix` and `GeneralImprovements`.
I was unable to use the shovel once when using only `FlashlightFix`, but I was able to use it again afterwards, but I was unable to record this footage.
After installing `GeneralImprovements`, the shovel was still unusable after launching the game.

My guess is that this is an issue with the item slot functionality

Video of using `FlashlightFix` and `GeneralImprovements` together
https://youtu.be/4KzntpW09Rs

The log for that video
[LogOutput.log](https://github.com/Shaosil/LethalCompanyMods-FlashlightFix/files/14443697/LogOutput.log)

**Comment by Shaosil** [comment]
This will hopefully be fixed in v1.1.2, which is going live shortly. Please let me know if not and I'll reopen this issue. Thanks for the info!
